app/services/search/query.rb

```diff
@@ -284,6 +296,22 @@ def self.licence_filter(licence)
       }
     end
 
+    def self.organisation_category_filter(organisation_category, boost: 2)
```

Comment:
should we extract the magic number `2` into a constant?

Reply:
I'm not sure this would be considered a magic number. I've mirrored the Elasticsearch domain - https://www.elastic.co/guide/en/elasticsearch/reference/6.x/mapping-boost.html where "boost" is a thing.